lib/config/config-file.js

```diff
@@ -313,7 +313,7 @@ function write(config, filePath) {
 function getBaseDir(configFilePath) {
 
     // calculates the path of the project including ESLint as dependency
-    var projectPath = path.resolve(__dirname, "../../../");
+    var projectPath = path.resolve(__dirname, "../../../../");
```

Comment:
This doesn't make sense to me. If this file hasn't moved, then why is this change necessary?